spec/index.html

```diff
@@ -17,7 +17,7 @@
         edDraftURI:           "https://w3c.github.io/sparql-entailment/spec/",
         testSuiteURI:         "https://w3c.github.io/rdf-tests/",
         shortName:            "sparql12-entailment",
-        copyrightStart:       "2008",
+        copyrightStart:       "2012",
```

Comment:
Does the copyright start change? I've left it as is in other specs, or was it simply wrong in this particular spec?

Reply:
This is a SPARQL 1.1 document. REC at 2013, FPWD sometime before.

2008 is SPARQL 1.0  (?probably should be FPWD date - 2008 is REC)